examples/configs/llm/vertexai/README.md

```diff
@@ -0,0 +1,22 @@
+# NeMo LLM Service Example
```

Comment:
This needs to be updated.

Reply:
Updated in commit [5319dbc5b89201e1e4d7046386183dbd7c3b4a12](https://github.com/NVIDIA/NeMo-Guardrails/pull/402/commits/5319dbc5b89201e1e4d7046386183dbd7c3b4a12).